packages/vscode-extension/package.nls.json

```diff
@@ -373,6 +373,7 @@
     "teamstoolkit.taskDefinitions.args.prerequisites.portsTitle": "Check ports availability for debug use.",
     "teamstoolkit.taskDefinitions.args.portOccupancy.title": "The port numbers to validate",
     "teamstoolkit.taskDefinitions.args.env.title": "The environment name.",
+    "teamstoolkit.taskDefinitions.args.expiration.title": "The tunnel will be deleted automatically if it's not used for a certain amount of time. This time is measured in seconds, and the default value is set to 3600 seconds.",
```

Comment:
if it's not used for a certain amount of time. This time is measured in seconds, and the default value is set to 3600 seconds.
->
If it has been inactive for 3600 seconds.

Reply:
THX! The expiration description has been updated.